test/test_manifest.json

```diff
@@ -2247,6 +2247,12 @@
        "lastPage": 4,
        "type": "eq"
     },
+    {  "id": "issue4227",
+      "file": "pdfs/coons-allflags-withfunction.pdf",
+      "md5": "c5f79c24bf9eb66698be0e4ecaa1bdf8",
+      "rounds": 1,
+      "type": "eq",
```

Comment:
You're right. Keep it like https://github.com/CodingFabian/pdf.js/blob/fixup-6303/test/test_manifest.json#L2331-L2336 for this PR as that is what we always use.